### DISCUSSION TITLE
请教一个关于API的问题

### DISCUSSION THREAD

**Discussion by CainLuo** [BTMuli/TeyvatGuide#102] [Q&A] [answered]
大佬，我装上你的App，发现是有请求 /game_record/app/genshin/api/character 这个API的，但是在代码上并没有搜索到，咋回事列？一头雾水。。。

**Comment by BTMuli** [comment, marked as answer]
https://github.com/BTMuli/TeyvatGuide/blob/master/src%2Fpages%2FUser%2FCharacters.vue#L12-L12
